### PR TITLE
Added missing tests

### PR DESCRIPTION
**Pull request by theyiwenliu**
Prerequisites for running tests (for your tensorflow Virtualenv):
- latest tensorflow and tensorboard, I am using 1.8.0 now
`pip install --upgrade tensorflow tensorboard`
- Install grpc and grpcio
`pip install grpc grpcio`

How to test:
- Python tests:
`bazel test tensorboard/...`

- Selenium:
`bazel test //tensorboard/functionaltests:core_test`

- Frontend unit tests related to Graph plugin changes:
`bazel run //tensorboard/plugins/graph/tf_graph_common/test`
open http://localhost:6006/tf-graph-common/test/index.html and `command + option + j` (if you are using Chrome) to see if it passed.

### CODE REVIEW

**Comment by theyiwenliu** [comment]
Ok now all tests passed!

File: tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html [hunk 1]
```diff
@@ -337,29 +337,29 @@
...
-    <div class="control-holder">
-      <div class="title">Upload</div>
-      <paper-button raised class="text-button upload-button"
-          on-click="_getFile">Choose File</paper-button>
-      <div class="hidden-input">
-        <input type="file" id="file" name="file" on-change="_updateFileInput" />
-      </div>
-    </div>
-  </template>
+  <!--<template is="dom-if" if="[[showSessionRunsDropdown]]">-->
```
Comment (kkasravi) [inline tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html]: any reason to keep the commented out section?

**Review by kkasravi — CHANGES_REQUESTED** [review]

File: tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html (hunk 1 above)
Comment (theyiwenliu) [reply tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html]: Not really, I'll remove them later 👍

Comment (theyiwenliu) [reply tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html]: Done!

**Review by kkasravi — APPROVED** [review]

**Pull request merged**